dashboard/models.py

```diff
@@ -273,10 +273,10 @@ def get_queryset(self):
     def get_course_resource_type(self, course_id):
         return self.get_queryset().get_course_resource_type(course_id)
 
-
 class Resource(models.Model):
+    id = models.AutoField(primary_key=True, verbose_name="Table Id")
     resource_type = models.CharField(max_length=255, verbose_name="Resource Type")
-    id = models.CharField(primary_key=True, max_length=255, verbose_name="Resource Id")
+    resource_id = models.CharField(blank=True, max_length=255, null=False, verbose_name="Resource Id")
```

Comment:
I think @jonespm  was talking about using `db_index=True` here, too, to help with performance. Probably makes sense to do this on `course_id`, too.

Reply:
https://docs.djangoproject.com/en/3.0/ref/models/fields/#db-index

Reply:
Do we need to allow `blank=True` for some reason?